src/components/ProfileDashboard.js

```diff
@@ -0,0 +1,466 @@
+        console.log("Profile:", body);
+        return body;
+    }
+
+    // Parses through the data for the user if there is existing data
+    const handleAPIUserProfile = (userID) => {
+        callAPIUserProfile(userID)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+
+                if (parsed.length !== 0) {
+                    setExistingAboutMe(parsed[0].aboutMe);
+                    setExistingYear(parsed[0].yearSemester);
+                    setExistingProgram(parsed[0].program);
+                    setExistingInterest(parsed[0].interest);
+                    setExistingCoop(parsed[0].coop);
+                }
+            })
+    }
+    
+    return (
+        // Displays the AppBar, Drawer, the Title, the fields and conditionals for the SAVE/EDIT button. 
+        <div className={classes.root}>
+            <AppBar position="fixed" className={classes.appBar}>
+                <Toolbar>
+                    <Typography variant="h3" noWrap>
+                        Profile
+                    </Typography>
+                </Toolbar>
+            </AppBar>
+            <Drawer
+                className={classes.drawer}
+                variant="permanent"
+                classes={{
+                    paper: classes.drawerPaper,
+                }}
+            >
+                <Toolbar />
+                <div className={classes.drawerContainer}>
+                    <List>
+                        <ListItem button component={Link} to="/profile">
+                            <ListItemText>
+                                <Typography variant="h6" fontWeight="bold" textAlign="center">
+                                    My Profile
+                                </Typography>
+                            </ListItemText>
+                        </ListItem>
+                        <ListItem button component={Link} to="/">
+                            <ListItemText>
+                                <Typography variant="h6" fontWeight="bold" textAlign="center">
+                                    Search Profiles
+                                </Typography>
+                            </ListItemText>
+                        </ListItem>
+                    </List>
+                </div>
+            </Drawer>
+            <Toolbar />
+            <Box sx={{ position: 'absolute', top: 100, left: "42%" }} >
+                <Card>
+                    <Card.Body>
+                        <h2> {firstName} {lastName}'s HonkedIn Profile</h2>
+                    </Card.Body>
+                </Card>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 240, left: "35%" }} >
+                <h5>About Me: </h5>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 225, left: "45%" }} >
+                <div>
+                    {loadProfile ? <h6> {existingAboutMe} </h6> : <form className={classes.root} noValidate autoComplete="off">
```

Comment:
You should be rendering the text like so:
```
            <Box sx={{ position: 'absolute', top: 240, left: "35%" }} >
                <h5>About Me: {existingAboutMe}</h5>
            </Box>
```

instead of:
```
            <Box sx={{ position: 'absolute', top: 240, left: "35%" }} >
                <h5>About Me: </h5>
            </Box>
```

doing so will align the text.